archivo.py

```
@@ -0,0 +1,7 @@
+# Archivo de prueba python
+
+def prueba():
+  asdasdasd
+  return cosa
```

Ruff (0.8.2)

Undefined name `cosa`

(F821)